sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterOptions.java

```diff
@@ -333,19 +302,7 @@ void internalBuildAndFreeze(ConfigProperties configProperties) {
         }
     }
 
-    /**
-     * Creates an Azure monitor metric exporter based on the options set in the builder. This
-     * exporter is an implementation of OpenTelemetry {@link MetricExporter}.
-     *
-     * <p>When a new {@link MetricExporter} is created, it will automatically start {@link
-     * HeartbeatExporter}.
-     *
-     * @param configProperties The OpenTelemetry configuration properties.
-     * @return An instance of {@link MetricExporter}.
-     * @throws NullPointerException if the connection string is not set on this builder or if the
-     * environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
-     */
-    public MetricExporter buildMetricExporter(ConfigProperties configProperties) {
+    MetricExporter buildMetricExporter(ConfigProperties configProperties) {
```

Comment:
Do we need these build methods? We no longer have the builder.

Reply:
These methods are used in the [AzureMonitorExporter](https://github.com/Azure/azure-sdk-for-java/blob/4de1c0ae0f9dad44afa29f90e6272df9365b4e2d/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporter.java#L64) class. They are no longer visible to users.  Please see https://github.com/Azure/azure-sdk-for-java/pull/42134#discussion_r1782718994